techniques/client-side-script/SCR40.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Using the CSS prefers-reduced-motion query in JavaScript to prevent motion</title>
+  <link rel="stylesheet" type="text/css" href="../../css/editors.css">
+</head>
+<body>
+
+<h1>Using the <abbr title="Cascading Style Sheets">CSS</abbr> <code>prefers-reduced-motion</code> query in JavaScript to prevent motion</h1>
+
+<section id="meta">
+  <h2>Metadata</h2>
+  <p id="id"></p>
+  <p id="technology"></p>
+  <p id="type"></p>
+</section>
+
+<section id="applicability">
+  <h2>When to Use</h2>
+  <p>
+    JavaScript animation which causes motion that is triggered by user interactions.
+  </p>
+</section>
+
+<section id="description">
+  <h2>Description</h2>
+  <p>The objective of this technique is to allow users to prevent animations (including motion animations) from being displayed on web pages, by using JavaScript to evaluate the <code class="language-css">prefers-reduced-motion</code> CSS Media Query.</p>
+  <p>Some users experience distraction or nausea from animated content. For example, if scrolling a page causes elements to move (other than the essential movement associated with scrolling the content, which is under the user's control) it can trigger vestibular disorders.</p>
+  <p>Media queries that selectively enable/disable JavaScript-driven animations based on operating system or user agent preferences allow users to prevent those symptoms.</p>
```

Comment:
```suggestion
  <p>Media queries that selectively enable/disable JavaScript-driven animations based on operating system or user agent preferences allow users to avoid those triggers.</p>
```